modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -37,7 +37,7 @@ When writing FCC files, note the following changes:
 ** `EC2` is now `AWS`
 ** `GCE` is now `GCP`
 +
-For more info, see the https://github.com/coreos/afterburn/blob/master/README.md[Afterburn readme].
+For more info, see the https://github.com/coreos/afterburn/blob/master/docs/usage/attributes.md[Afterburn documentation].
```

Comment:
What about linking this to the upcoming 4.4.1 release to ensure the target stays in place?

The line of reasoning is: it's easier to find a resource that moved somewhere else (or got updated) if you are at least getting an handle to when/where it used to exist.

Reply:
Wouldn't it make more sense to prefer linking to current documentation, rather than optimizing for the rare case that we have to update a dead link?

Reply:
Eh, I don't agree it's a rare case, but I agree that I should stop fighting windmills at this point :)